backend/prisma/schema.prisma

```diff
@@ -19,55 +19,51 @@ enum UserType {
 }
 
 model User {
-    id Int @id @default(autoincrement())
-    username String 
-    email String 
-    password String
-    userType UserType
-    attendee Attendee?
-    salt String 
-    dateJoined DateTime
-    profilePicture String?
-    societies Society[]
- }
-
-model Attendee {
-  id Int @id @default(autoincrement())
-  user User @relation(fields: [userId], references: [id])
-  userId Int @unique
-  name String
-  keywords Keyword[]
-  societies Society[]
+  id             Int       @id @default(autoincrement())
+  username       String
+  email          String
+  password       String
+  userType       UserType
+  salt           String
+  dateJoined     DateTime
+  profilePicture String?
+  societies      Society[] @relation(name: "membertosociety")
+  keywords       Keyword[]
+  adminsocieties Society[]
+  events         Event[]
 }
 
 model Society {
-  id Int @id @default(autoincrement())
-  admin User @relation(fields: [userId], references: [id], onDelete: Cascade)
-  userId Int
-  name String @unique
-  events Event[]
-  members Attendee[]
-  discordId String? @unique
+  id             Int     @id @default(autoincrement())
+  admin          User    @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId         Int
+  name           String  @unique
+  events         Event[]
+  members        User[]  @relation(name: "membertosociety")
+  discordId      String? @unique
   profilePicture String?
 }
 
 model Event {
-  numId Int @id @default(autoincrement())
-  banner String
-  name String
+  numId         Int       @id @default(autoincrement())
+  banner        String
+  name          String
   startDateTime DateTime
-  endDateTime DateTime
-  location String
-  description String
-  societyId Int
-  society Society @relation(fields:[societyId], references: [id], onDelete: Cascade)
-  keywords Keyword[]
+  endDateTime   DateTime
+  location      String
+  description   String
+  societyId     Int
+  society       Society   @relation(fields: [societyId], references: [id], onDelete: Cascade)
+  keywords      Keyword[]
+  attendees     User[]
+  numAttendees  Int       @default(0)
```

Comment:
unless this is something prisma specifically suggests i would avoid this personally. when we add or remove attendees we'd have to update this value as well and that either sounds:

1. bound to cause concurrency problems, or
2. issues where one is updated and not the other by mistake